src/views/docs-view/server.ts

```diff
@@ -260,27 +262,38 @@ export function getStaticGenerationFunctions<
 			}
 
 			/**
-			 * Constructs the levels of nav data used in the `Sidebar` on all
-			 * `DocsView` pages.
+			 * Constructs the base sidebar level for `DocsView`.
+			 */
+			const docsSidebarLevel: SidebarProps = {
+				backToLinkProps: {
+					text: `${product.name} Home`,
+					href: `/${product.slug}`,
+				},
+				levelButtonProps: {
+					levelUpButtonText: `${product.name} Home`,
+				},
+				menuItems: navDataWithFullPaths as EnrichedNavItem[],
+				// If the title is not hidden for this rootDocsPath, include it
+				title: BASE_PATHS_TO_NAMES[basePath] || product.name,
```

Comment:
Can we retain the `TODO` comment that was present before? Looks like this might be an accidental copy/paste from the new `hideSidebarTitle` block

Reply:
Absolutely, great catch, this was a slip on my part (exactly as you suspected!). I've added it back 👍